### DISCUSSION TITLE
Swagger 제거 후 발생한 commons-lang3 의존성 문제

### DISCUSSION THREAD

**Discussion by nayonsoso** [solid-connection/solid-connect-server#114] [트러블 슈팅]
## 문제 상황
swagger 관련 코드를 제거하고, api를 hoppscotch로 관리하기로 했다.
그래서 swagger 의존성인 `springdoc-openapi-starter-webmvc-ui`를 없애고, 어플리케이션을 돌렸다.
그랬더니 아래와 같은 예외가 발생하며, 실행이 되지 않았다.
`org.apache.commons.lang3` 와 스웨거는 너무 관련이 없어 보이고,.. 이 의존성은 애초에 존재하지 않았는데 이게 무슨 일일까😳

![image](https://github.com/user-attachments/assets/b51c09ff-f6ea-45fb-bd74-7eabcf473984)

## 문제 원인
`./gradlew dependencies --configuration compileClasspath` 
명령어를 사용하면 gradle 의존성의 트리를 볼 수 있다.
확인해보니, 전혀 연관이 없을 것처럼 보이는 두 라이브러리가 의존 관계에 있었다.

![image](https://github.com/user-attachments/assets/1524140a-a0e2-499d-b185-e45bb09840c1)

## 문제 해결
swagger 에 대한 의존성은 없애고, org.apache.commons:commons-lang3 의존성만 남겨 해결했다.

**Comment by leesewon00** [comment]
hoppscotch에 대해서도 간략히 설명해주시면 좋을 것 같습니다!
